src/web/components/Card/components/CardFooter.tsx

```diff
@@ -1,29 +1,68 @@
 import React from 'react';
-import { css, cx } from 'emotion';
+import { css } from 'emotion';
+
+import { GuardianLines } from '@frontend/web/components/GuardianLines';
+
+import { LinesWrapper } from './LinesWrapper';
 
 type Props = {
-    children: JSXElements;
-    designType?: DesignType;
+    designType: DesignType;
+    age?: JSX.Element;
+    mediaMeta?: JSX.Element;
+    commentCount?: JSX.Element;
 };
 
-const footerStyles = css`
+const spaceBetween = css`
     display: flex;
     justify-content: space-between;
     align-items: center;
 `;
 
-const mediaFooterStyles = css`
-    flex-direction: row-reverse;
-    padding: 0 5px 5px 5px;
+const flexEnd = css`
+    display: flex;
+    justify-content: flex-end;
 `;
 
-export const CardFooter = ({ children, designType }: Props) => (
-    <footer
-        className={cx(
-            footerStyles,
-            designType === 'Media' && mediaFooterStyles,
-        )}
-    >
-        {children}
-    </footer>
-);
+export const CardFooter = ({
+    designType,
+    age,
+    mediaMeta,
+    commentCount,
+}: Props) => {
+    if (designType === 'Comment') {
+        return (
+            <footer className={spaceBetween}>
+                {age}
+                <LinesWrapper>
+                    <GuardianLines pillar="opinion" count={4} />
+                </LinesWrapper>
+                {commentCount}
+            </footer>
+        );
+    }
+
+    if (designType === 'Media') {
```

Comment:
wouldnt it be interesting to have designType as a ENUM and it to compare things like this?

Reply:
Sure, you could. I guess the benefit to these types of patterns is reduced when you have typescript though as it's impossible to make a typo here